### ISSUE TITLE
Where's the loss code for the stage2 finetune?

### ISSUE THREAD

**Issue by chaixinning** [zhenshij/arbitrary-scale-diffusion#5]

**Comment by chaixinning** [comment]
In equation 7 of your paper, how do you compute the fake x_0 during each training step? To compute a x_0 during training, do you use a single diffusion step or multiple diffusion steps?
